### PR TITLE
Paola checklist

### PR DESCRIPTION
**Pull request by paolap**
I've only added the checklists from the Clex training as images to provide a starting point and add some text in the same page, which used to be empty.
There's no changes to anything else, if you want you might be able to at least view a copy of the original slides in our google folder: https://docs.google.com/presentation/d/1fh4jiHwCU21LosdYoj0OyhGjZ6I10NEhkIEItXZbvCE/edit?usp=drive_web&ouid=111871013972655065281

I put them there as it might be easier to adapt these and then download them as images than restart from zero.

Eventually, we want the workflow implicitly suggested by the guidelines to match what we suggest here

### CODE REVIEW

**Review by hot007 — APPROVED** [review]
I really like these checklists, Nice to include them here.
I wonder if we should also make an 'unchecked' version available for people to download and work through themselves?

File: Governance/create/create-checklists.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# Checklists for data management in the project lifecycle
\ No newline at end of file
+# Checklists for data management in the project lifecycle
+
+These checklists represent what a dataset evolution might look like at different stages of a project.
+Not all of these action will have the same relevance for different projects and some datasets might need further specific actions.
+The checklists highlight in `yellow` what should be accomplish at each stage and in `blue` what should have already happened.
```
Comment (AliciaTak) [inline Governance/create/create-checklists.md:5]: The checklist is great, thanks!
The last sentence "The checklists highlight in `yellow` what should be accomplish at each stage and in `blue` what should have already happened." is a bit unclear, but I imagine this is only a draft version. I think you probably mean that `blue` is essential, and `yellow` is essential, too, but needs to be updated along the way?
Also, what are the `black` ones?

**Review by AliciaTak — APPROVED** [review]

File: Governance/create/create-checklists.md (hunk 1 above)
Comment (hot007) [reply Governance/create/create-checklists.md:5]: @paolap can you please address @AliciaTak 's comments if you think it's useful (ie explain the colour scheme), and merge? I'll progress with other issues tomorrow hopefully once this is merged.

Comment (paolap) [reply Governance/create/create-checklists.md:5]: Apologies it took so long to get back to this, so the answer are "yeas this is a draft" to start, we might want to modify the lists these are just images of the slides I had for a CLEX training, inserted in the file as they were.
The colour are explained in the create-checklist.md file, in particular this sentence:

"The checklists highlight in `yellow` what should be accomplish at each stage and in `blue` what should have already happened."

But let me know if it's not clear. 
@hot007 I like the idea of having downloadable lists so I'll see if there's a an easy way to link to them.

Comment (paolap) [reply Governance/create/create-checklists.md:5]: @AliciaTak my apologies, I misread your comment and thought that you didn't see the note of what yellow and blue mean. I tried to make more clear that the checklist is presented in 3 different views, one for each stage of the project. Hopefully it is clearer, we can still improve it later. We could potentially write a bit more about this, as well as change the lists.
I also included a word version of the checklist in the repository, and the same and google doc version on the drive folder, these are linked into the page itself.

**Pull request merged**